config/config.go

```diff
@@ -28,9 +28,9 @@ func Get() (*Config, error) {
 	}
 
 	if cfg.Debug {
-		cfg.PatternLibraryAssetsPath = "http://localhost:9000/dist"
+		cfg.PatternLibraryAssetsPath = "http://localhost:9001/dist/assets"
```

Comment:
Incorrect port number
```suggestion
		cfg.PatternLibraryAssetsPath = "http://localhost:9002/dist/assets"
```